### DISCUSSION TITLE
Heroku deployment description seems to be outdated

### DISCUSSION THREAD

**Discussion by iamNoah1** [danny-avila/LibreChat#1702] [Feature Requests & Suggestions]
https://docs.librechat.ai/deployment/heroku.html seems not to be up to date. 

When I create the suggested dockerfiles, at least when trying to run the api one I get `please run this from the root directory`. Also I am not sure if we really need to create new Dockerfiles or if we can use the present ones instead.

**Comment by danny-avila** [comment]
indeed heroku does need an update. it seems to benefit off deploying 2 separate docker images. 

This is essentially what the `Dockerfile.multi` file does, so maybe it can be simplified without needing to create a custom dockerfile for heroku.
